modules/nw-infw-operator-rules-object.adoc

```diff
@@ -110,6 +110,7 @@ spec:
           icmpType: 128 #ICMPV6 Echo request
       action: Deny
 ----
+<1> The `ingress_firewall_label` must match the `nodeSelector` label that you specified in the `ingressfirewallconfig` CR.
```

Comment:
Not necessarily match the same label in the config here is clarification example assume cluster with 5 nodes and u wanted fw on only 4 nodes so you label 4 nodes with do-fw = true in the configuration cr now I need to run group A rules on two nodes and group B on the other two to achieve this using node selector in fw rule object so creating two rules objecs a ne with group-a label and the other with group-b obviously fw will kick in if the node involved in fw had config and rules labels if any is missing then configuration is noop

Reply:
I don't think I follow what you are saying. Are you saying the `nodeSelector` label does not have to match the `nodeSelector` label that you specified in the `ingressfirewallconfig` CR; rather, the label needs to match the `nodeSelector` labels applied to the specific nodes with which you wish to apply the `ingressfirewallconfig` CR. This would allow you to apply different rules using the CR that could target different groups of nodes, which I think is what you are describing.

Reply:
Correct